src/components/LPolyline.vue

```diff
@@ -0,0 +1,45 @@
+<script>
+import { onMounted, ref, inject } from "vue";
+import { remapEvents, propsBinder } from "../utils.js";
+import { props, setup as polylineSetup } from "../functions/polyline";
+
+/**
+ * Polyline component, lets you add and personalize polylines on the map
+ */
+export default {
+  name: "LPolyline",
+  props,
+  setup(props, context) {
+    const leafletRef = ref({});
+    const ready = ref(false);
+
+    const addLayer = inject("addLayer");
+
+    const { options, methods } = polylineSetup(props, leafletRef, context);
+
+    onMounted(async () => {
+      const { polyline, DomEvent, setOptions } = await import(
+        "leaflet/dist/leaflet-src.esm"
+      );
+
+      leafletRef.value = polyline(props.latLngs, options);
+
+      const listeners = remapEvents(context.attrs);
+      DomEvent.on(leafletRef.value, listeners);
+
+      propsBinder(methods, leafletRef.value, props, setOptions);
+
+      addLayer({
+        ...props,
+        ...methods,
+        leafletObject: leafletRef.value,
+      });
+      ready.value = true;
+    });
+    return { ready };
```

Comment:
What is the reason to return the `ready` value, or in fact define it at all? As far as I can see it's never used in the polyline logic, and it's definitely not used in the render template either.

Reply:
@mikeu you are right! The component in the v2 was using it but I thought it did not made much sense, upon further thought I realise that a `polyline` is infact a layer... so you may want to nest something in it, addressing this now!

Reply:
Fixed :)